### PR TITLE
#1310 Migrate sql querys to xorm builder

### LINKED ISSUE #1310: migrate more sql querys to xorm
migrate code like e.g. https://github.com/woodpecker-ci/woodpecker/pull/1296/files#diff-a3003b46215d34362e75d779f7e5b3fe234547af38afc3d21b7402f3a256636bL51-L61

to use the xorm builder or xorm instead

### PR DESCRIPTION
**Pull request by bitethecode**
close  #1310

### CODE REVIEW

File: server/model/feed.go [hunk 1]
```diff
@@ -19,24 +19,24 @@ package model
...
+	Commit   string `json:"commit,omitempty"        xorm:"feed_pipeline_commit"`
+	Branch   string `json:"branch,omitempty"        xorm:"feed_pipeline_branch"`
+	Ref      string `json:"ref,omitempty"           xorm:"feed_pipeline_ref"`
+	Refspec  string `json:"refspec,omitempty"       xorm:"feed_pipeline_refspec"`
+	Remote   string `json:"remote,omitempty"        xorm:"feed_pipeline_remote"`
+	Title    string `json:"title,omitempty"         xorm:"feed_pipeline_title"`
+	Message  string `json:"message,omitempty"       xorm:"feed_pipeline_message"`
+	Author   string `json:"author,omitempty"        xorm:"feed_pipeline_author"`
+	Avatar   string `json:"author_avatar,omitempty" xorm:"feed_pipeline_avatar"`
+	Email    string `json:"author_email,omitempty"  xorm:"feed_pipeline_email"`
```
Comment (anbraten) [inline server/model/feed.go:41]: What a crazy model. I guess we should clean it up at some point as well.

Comment (6543) [reply server/model/feed.go:41]: @anbraten well it act's like a `view`

Comment (6543) [reply server/model/feed.go:41]: and I prefixed it, so if we refactor stuff it wont break

Comment (anbraten) [reply server/model/feed.go:41]: Sure. I was mainly talking about all the different properties as I guess most of them are not even part of our UI atm.

Comment (6543) [reply server/model/feed.go:41]: well in an dedicated refactor we can remove all not needed once and expose those who do make sense to the ui

**Pull request merged**